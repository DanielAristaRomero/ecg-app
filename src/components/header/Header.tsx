export const Header = ({ titulo = "NombreSeccion" }) => {
  return (
    <div>
      <div className="relative md:hidden">
        <div className="flex justify-center absolute w-full top-3">
          <h4 className="text-white text-2xl">{titulo}</h4>
        </div>
      </div>
      <div className="h-[150px] overflow-hidden">
        <svg
          className="h-full w-full"
          viewBox="0 0 500 150"
          preserveAspectRatio="none"
        >
          <path
            d="M-0.27,54.78 C113.15,74.52 364.27,35.05 500.84,59.72 L500.00,0.00 L0.00,0.00 Z"
            style={{ stroke: "none", fill: "#08f" }}
          ></path>
        </svg>
      </div>
    </div>
  );
};
