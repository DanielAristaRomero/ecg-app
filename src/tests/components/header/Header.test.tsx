import { shallow } from "enzyme";
import { Header } from "../../../components/header/Header";

describe("Pruebas sobre Header.tsx", () => {
  test("debe rendirizar el componente correctamente", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
