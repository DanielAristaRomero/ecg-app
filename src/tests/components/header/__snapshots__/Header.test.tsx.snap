// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre Header.tsx debe rendirizar el componente correctamente 1`] = `ShallowWrapper {}`;
